src/components/Articles/Articles.js

```diff
@@ -3,20 +3,28 @@ import React from 'react';
 import './Articles.css';
 
 function Articles(props) {
+  let articleData;
 
-    const { searchResults } = props;
-
-    return (
-        <main className='articles-container'>
-            <div className='articles'>
-                <h3 className='article-header'>Test</h3>
-                <a href='#' className='article-link'>
-                <p className='article-snippet'>Test(s) or TEST may refer to: 
-                Test (assessment), an assessment intended to measure the 
-                respondents' knowledge or other abilities List of tests</p></a>
+    if (props.searchResults.length) {
+        articleData = props.searchResults.map((obj) => (
+            <div className='articles' key={obj.pageid}>
+              <h3 className='article-header'>{obj.title}</h3>
+              <a 
+              href={`https://en.wikipedia.org/?curid=${obj.pageid}`} 
+              className='article-link'
+              target='_blank'
+              rel='noopener noreferrer'>
+              <p className='article-snippet'>
+              {obj.snippet.replace(/<\/?[^>]+>/gi, '')}
+              </p></a>
             </div>
-        </main>
-    )
-}
+        ));
+        } else {
```

Comment:
extra indentation indentation.

Highly recommend checking out `eslint`, and `airbnb linting standards` - if you manage to integrate both, will help you become a much better programmer